flink-python/pyflink/datastream/connectors/cassandra.py

```diff
@@ -126,14 +132,340 @@ def if_not_exists(self, enabled: bool) -> 'MapperOptions':
         return self
 
 
+# ---- Classes introduced to construct the ClusterBuilder ----
+
+
+class LoadBalancingPolicy(object):
+    """
+    The policy that decides which Cassandra hosts to contact for each new query.
+
+    The LoadBalancingPolicy is informed of hosts up/down events. For efficiency purposes, the policy
+    is expected to exclude down hosts from query plans.
+    """
+
+    def __init__(self, j_load_balancing_policy):
+        self._j_load_balancing_policy = j_load_balancing_policy
+
+    @staticmethod
+    def dc_aware_round_robin_policy() -> 'LoadBalancingPolicy':
```

Comment:
Could we also investigate whether it's possible to support the following policy: 

- WhiteListPolicy: See WhiteListPolicy.ofHosts for more details on how to create WhiteListPolicy
- HostFilterPolicy: See HostFilterPolicy.fromDCWhiteList and HostFilterPolicy.fromDCBlackList for more details
- LatencyAwarePolicy
- TokenAwarePolicy

Reply:
What about fixing this in a separate PR as this PR is already very big？